h/static/scripts/widget-controller.js

```diff
@@ -314,7 +314,7 @@ module.exports = function WidgetController(
 
   // When a direct-linked annotation is successfully anchored in the page,
   // focus and scroll to it
-  $rootScope.$on(events.ANNOTATIONS_SYNCED, function (event, tags) {
+  $scope.$on(events.ANNOTATIONS_SYNCED, function (event, tags) {
```

Comment:
Unfortunately neither of these two subscriptions here get triggered because of the use of `$rootScope.$emit` rather than `$rootScope.$broadcast` for emitting events when messages are received from the page.

To test handling of these two events manually:

1. Load the page with "#annotations:<annotation ID>" in the URL and verify that the direct-linked annotation is selected
2. Select an annotation, so that the sidebar shows only the selection, then select text in the page and create a new annotation. The selection should be cleared so that the annotation card for the new annotation is visible